### ISSUE TITLE
PRINT does not work on Windows in the script specified on the command line

### ISSUE THREAD

**Issue by meshpoint** [metaeducation/ren-c#49]
Compiled with mingw-w64, tested on Windows XP and Wine.

    >type test-stdio.r
    REBOL [Title: "Test PRINT and ASK"]
    print "PRINT works"
    ; use WRITE in case PRINT doesn't work
    write %out.txt ask "ASK: "
    
    >r3 test-stdio.r
    user input
    
    >type out.txt
    user input

As you can see, the input part of ASK works, while the output doesn't. Interestingly, PRINT works if the script is invoked any other way.

    >r3 --do "do %test-stdio.r"
    PRINT works
    ASK: user input

**Comment by hostilefork** [comment]
Thanks for the report.  Since you haven't said explicitly that I shouldn't, I'll get to debugging it now...

**Comment by earl** [comment]
I can confirm the problem (cross-compiled current master with 32-bit mingw-w64 gcc 5.2.0 to Windows/x86, tested via Wine).

**Comment by earl** [comment]
I pushed a fix to a [renc-fix-49-windows-stdout](https://github.com/earl/r3/tree/renc-fix-49-windows-stdout) ([diff w/o ws changes](https://github.com/metaeducation/ren-c/compare/master...earl:renc-fix-49-windows-stdout?w=1)) branch. Basically, this removes remnants of Windows console attachment magic which was left-over from the rebase on top of Atronix/Saphirion. The gist is, that there's a lot of attempted workarounds for [not being able to get good hybrid console-mode/GUI-mode behaviour on Win32]. As Ren/C doesn't have a GUI dialect anyway, I removed most of those pieces. If interested, see the commit messages for details (beware: there's an essay buried in there ...).

I'd be very grateful for test builds and feedback if things (console & script execution) don't work as expected. I'll merge it tomorrow, unless I hear of major breakage. But then, PRINT not working is already about as broken as it gets :)
